client/src/modules/required_inventory_scans/modals/search.modal.html

```diff
@@ -24,12 +24,27 @@
             <bh-clear on-clear="$ctrl.clear('depot_uuid')"></bh-clear>
           </bh-depot-select>
 
+          <div class="form-group">
+            <label class="control-label" translate>FORM.LABELS.REFERENCE_NUMBER_OPTIONAL</label>
+            <bh-clear on-clear="$ctrl.clear('reference_number')"></bh-clear>
+            <input ng-model="$ctrl.searchQueries.reference_number"
+              class="form-control"
+              type="text"
```

Comment:
Should we put a max limit on this field?

Reply:
Not now.  We don't know how long these will be since they depend on what external partners want to use.  See more notes on this in the comments below.